process/process_windows.go

```diff
@@ -882,7 +882,8 @@ func getFromSnapProcess(pid int32) (int32, int32, string, error) { //nolint:unpa
 	defer windows.CloseHandle(snap)
 	var pe32 windows.ProcessEntry32
 	pe32.Size = uint32(unsafe.Sizeof(pe32))
-	if err = windows.Process32First(snap, &pe32); err != nil {
+	err = windows.Process32First(snap, &pe32)
+	if err != nil {
```

Comment:
Is this change perhaps intended to a code like this?
```suggestion
	if err := windows.Process32First(snap, &pe32); err != nil {
```

Reply:
Hmm, I think the original code might be incorrect. It was probably supposed to use `if err :=`.

I'm also curious why [this code](https://github.com/shirou/gopsutil/pull/1856/files/cf5debe6a1b2d1a705cc557ef0c57857cdf03dfd#diff-ae26c3608d9221e521907319158f351ef1a80515c3dc5b2e780c5eda6758f5a8R894) wasn't detected.